From 5b63f2aaccc3115e53d7f43dfd26ddabb1ae3273 Mon Sep 17 00:00:00 2001
From: Devarsh Thakkar <devarsht@xilinx.com>
Date: Wed, 25 Jan 2017 18:40:34 +0530
Subject: [PATCH 24/24] Link allocator lib to libgstomx.so

- Link libgstallocator to libgstomx.so
  as we have used dmabuf allocator

Signed-off-by: Devarsh Thakkar <devarsht@xilinx.com>
Upstream Status: Pending
---
 omx/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/omx/Makefile.am b/omx/Makefile.am
index 62f89c8..c087c25 100644
--- a/omx/Makefile.am
+++ b/omx/Makefile.am
@@ -94,6 +94,7 @@ libgstomx_la_LIBADD = \
 	-lgstaudio-@GST_API_VERSION@ \
 	-lgstpbutils-@GST_API_VERSION@ \
 	-lgstvideo-@GST_API_VERSION@ \
+	-lgstallocators-@GST_API_VERSION@ \
 	$(GST_BASE_LIBS) \
 	$(GST_LIBS) \
 	$(GMODULE_NO_EXPORT_LIBS)
-- 
2.7.4

